bip-0078.mediawiki

```diff
@@ -268,7 +275,6 @@ The sender should check the payjoin proposal before signing it to prevent a mali
 *** Verify the PSBT input is finalized
 *** Verify that <code>non_witness_utxo</code> or <code>witness_utxo</code> are filled in.
```

Comment:
This check is antithetical to PSBT IMO and may be vestigial of a specific PSBT signer in C#. It makes a pain for other signers in LND and BDK which search for places they can sign by script. Perhaps it's out of scope of this PR but I'd appreciate feedback either way.